doc/README.md

```diff
@@ -32,9 +32,9 @@ reference manual requires Python 3, and the following Python packages:
 
   - sphinx >= 4.5.0
   - sphinx_rtd_theme >= 1.0.0
-  - beautifulsoup4 >= 4.0.6
+  - beautifulsoup4 >= 4.8.2
   - antlr4-python3-runtime >= 4.7.1
-  - pexpect >= 4.2.1
+  - pexpect >= 4.6.0
```

Comment:
I guess those are the versions in ubuntu 20.04?

Reply:
Yes